Packs/CommonPlaybooks/Playbooks/playbook-Ticket_Management_-_Generic.yml

```diff
@@ -0,0 +1,703 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: bc8a5228-c2b8-46af-8e21-d7687a8edffe
+    type: playbook
+    task:
+      id: bc8a5228-c2b8-46af-8e21-d7687a8edffe
+      version: -1
+      name: Zendesk - Ticket Management
+      description: '`Zendesk - Ticket Management` allows you to open new tickets or update comments to the existing ticket using the playbook''s tasks. '
+      playbookName: Zendesk - Ticket Management
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      Action:
+        complex:
+          root: Action
+      CommentToAdd:
+        complex:
+          root: inputs.CommentToAdd
+      ZendeskAssigne:
+        complex:
+          root: inputs.ZendeskAssigne
+      ZendeskCollaborators:
+        complex:
+          root: inputs.ZendeskCollaborators
+      ZendeskPriority:
+        complex:
+          root: inputs.ZendeskPriority
+      ZendeskRequester:
+        complex:
+          root: inputs.ZendeskRequester
+      ZendeskStatus:
+        complex:
+          root: inputs.ZendeskStatus
+      ZendeskSubject:
+        complex:
+          root: inputs.ZendeskSubject
+      ZendeskTags:
+        complex:
+          root: inputs.ZendeskTags
+      ZendeskTicketID:
+        complex:
+          root: parentIncidentContext
+          accessor: ZendeskTicketID
+      ZendeskType:
+        complex:
+          root: inputs.ZendeskType
+      addCommentPerEndpoint:
+        complex:
+          root: inputs.addCommentPerEndpoint
+      description:
+        complex:
+          root: inputs.description
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 66b87a8c-c5a1-489c-8cef-ea4c198faa4e
+    type: title
+    task:
+      id: 66b87a8c-c5a1-489c-8cef-ea4c198faa4e
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 870,
+          "y": 950
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 159abeff-fe4e-42fc-8be0-67f4ba0f245b
+    type: regular
+    task:
+      id: 159abeff-fe4e-42fc-8be0-67f4ba0f245b
+      version: -1
+      name: Set Zendesk Ticket ID
+      description: commands.local.cmd.set.parent.incident.context
```

Comment:
Maybe add a more informational description explaining the if-then-else check.

Reply:
Done.